File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/13. Validate2/Python/Response.py
#!/python27/python

import cgi;


f = cgi.FieldStorage()
fname = f.getvalue('fname')
lname = f.getvalue('lname')
phone1 = f.getvalue('phoneArea')
phone2 = f.getvalue('phonePrefix')
phone3 = f.getvalue('phoneFour')
email = f.getvalue('email')


print("Content_type: 'text/html' \n\n")

print('''
<br>
First Name: {} <br>
Last Name: {} <br>
Phone Number: {}-{}-{} <br>
Email: {} <br>
'''.format(fname, lname, phone1, phone2, phone3, email))

File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/12. Validate/PHP/Response.php
<?php

$fname = $_REQUEST['fname'];
$lname = $_REQUEST['lname'];
$phone1 = $_REQUEST['phoneArea'];
$phone2 = $_REQUEST['phonePrefix'];
$phone3 = $_REQUEST['phoneFour'];
$email = $_REQUEST['email'];

print(
"
First Name: $fname<br>
Last Name: $lname<br>
Phone Number: $phone1-$phone2-$phone3<br>
Email: $email
");


?>